### PR TITLE
Under construction & new footer

### PR DESCRIPTION
**Pull request by dugdaniels**
Implementing the bounty at https://app.clarity.so/cabin/work/219 

Includes

- Building out an 'under construction' component
- Adding placeholders for upcoming pages
- Reworking the nav and footer
- Adding newsletter subscription form

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -1,90 +1,244 @@
...
 export default function Footer() {
+  const [email, setEmail] = useState("");
+  const [error, setError] = useState("");
+  const [confirmed, setConfirmed] = useState(false);
+
+  const onSubmit = async () => {
+    setConfirmed(false);
+    setError("");
+    try {
+      await axios.post("/api/convertkit", {
```
Comment (dugdaniels) [inline components/Footer.js:131]: @dvargas92495 Do you have access to ConvertKit? Need to get this hooked up.

Comment (dvargas92495) [reply components/Footer.js:131]: I do, though I see three forms. Gonna dm you what I see and the form Ids

File: components/Footer.js [hunk 2]
```diff
@@ -1,90 +1,244 @@
...
+      <Link href="/visit">
+        <a>Visit</a>
+      </Link>
+      <a target="_blank" rel="noreferrer" href="https://creators.mirror.xyz/">
+        Media
+      </a>
+    </>
+  ); 
+
+  const products = (
```
Comment (dvargas92495) [inline components/Footer.js:159]: we could probably be fancy here and use `getServerSideProps` to get this dynamically

File: components/UnderConstruction.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          <HeroFrame src={constructionImage} alt="A person buiding a cabin" />
+        </Content>
+      </Wrapper>
+    </Container>
+  );
+}
+
+export async function getServerSideProps() {
+  return { props: {} };
+}
```
Comment (dvargas92495) [inline components/UnderConstruction.js:24]: this function is unused when it's not in `/pages`

**Review by dvargas92495 — COMMENTED** [review]
Following up on CK in Discord

**Review by dvargas92495 — APPROVED** [review]
🎉

**Pull request merged**